jobsboard/jobs/models.py

```diff
@@ -26,3 +27,6 @@ class Job(TimeStampedModel):
     url = models.URLField()
     is_featured = models.BooleanField(default=False)
     employment_type = models.IntegerField(choices=EMPLOYMENT_TYPE, default=1)
+    location = models.CharField(max_length=256, blank=True)
+    email = models.CharField(max_length=100, blank=True)
+    expiry = models.DateTimeField(default=timezone.now() + timezone.timedelta(days=30))
```

Comment:
Please verify that every time you create a new `Job` instance, it reflects the time at `object creation` and `not the time that models.py was read by the interpreter` - potentially only once when the app is started).